### ISSUE TITLE
Improving the UI of the landing page and about section

### ISSUE THREAD

**Issue by shivani-tripurari** [akshitagupta15june/Moksh#278]
I would like to the the overall ui of home and about section.
Here is my design.
[Moksh_new.pdf](https://github.com/akshitagupta15june/Moksh/files/11528060/Moksh_new.pdf)

Hope this helps!

**Comment by akshitagupta15june** [comment]
can u please join discord channel for discussion on UI
awesome work

**Comment by shivani-tripurari** [comment]
I'm already in the Moksh's discord channel.
Okay then i'll have a discussion regarding ui there and if you approve i'll start working.

**Comment by mnik7044** [comment]
Yes and it's a request from our side to not start working on any issue before getting assigned.

**Comment by mnik7044** [comment]
Yes and it's a request from our side to not start working on any issue before getting assigned.

**Comment by mnik7044** [comment]
Yes and it's a request from our side to not start working on any issue before getting assigned.

**Comment by prathamesh-borse** [comment]
Hi, @shivani-tripurari are u still working on this issue? If so can you please tell what are the issues are resolved and what are the issues are remaining.!

**Comment by prathamesh-borse** [comment]
Hi, @shivani-tripurari are u still working on this issue? If so can you please tell what are the issues are resolved and what are the issues are remaining.!

**Comment by shivani-tripurari** [comment]
I'm working on other designs as the admins did not confirm about the final design
